libraries.json

```diff
@@ -753,5 +753,10 @@
     "github": "corbella83/PopKorn",
     "category": "DI",
     "maven": "https://repo1.maven.org/maven2/cc/popkorn/popkorn/"
+  },
+  {
+    "github": "dss99911/kotlin-simple-architecture",
+    "category": "Architecture",
+    "maven": "https://dl.bintray.com/hyun/kotlin-simple-architecture/kim/jeonghyeon/"
```

Comment:
at path `https://dl.bintray.com/hyun/kotlin-simple-architecture/kim/jeonghyeon/` not exist file `maven-metadata.xml`.

here should be path to core artifact, like 
https://dl.bintray.com/hyun/kotlin-simple-architecture/kim/jeonghyeon/kotlin-simple-api-client/
https://dl.bintray.com/hyun/kotlin-simple-architecture/kim/jeonghyeon/kotlin-simple-api-backend/
https://dl.bintray.com/hyun/kotlin-simple-architecture/kim/jeonghyeon/kotlin-simple-api-annotation/
https://dl.bintray.com/hyun/kotlin-simple-architecture/kim/jeonghyeon/kotlin-simple-architecture/
https://dl.bintray.com/hyun/kotlin-simple-architecture/kim/jeonghyeon/kotlin-simple-architecture-annotation/
https://dl.bintray.com/hyun/kotlin-simple-architecture/kim/jeonghyeon/kotlin-simple-architecture-client/
https://dl.bintray.com/hyun/kotlin-simple-architecture/kim/jeonghyeon/mpp-test/

just add this paths and script will resolve targets and kotlin version from this artifacts. 
with current maven path library will be removed by script because can't fetch info from maven.

Reply:
Thanks for the review.
I was confused how to add multiple artifact path.
I added 4 path to two category 'Architecture' and 'Network'.
Kindly review again.